src/material/datepicker/month-view.ts

```diff
@@ -259,7 +265,7 @@ export class MatMonthView<D> implements AfterContentInit, OnDestroy {
         // rest of the range logic, because focus may have moved outside the calendar body.
         if (this._matCalendarBody._previewEnd > -1) {
           this.selectedChange.emit(null);
-          this._userSelection.emit();
+          this._userSelection.emit({value: null, event});
```

Comment:
We can't, because `_dateSelected` has some extra parsing logic that doesn't handle `null`. It has been like this forever, I just updated it to match the new signature.